### ISSUE TITLE
Hyper-parameters settings？

### ISSUE THREAD

**Issue by David-19940718** [bellymonster/Weighted-Soft-Label-Distillation#8]
Hi, thank u for your nice job! Can you give me advice about how to set hyper-param of temperature coefficient and loss weight?

**Comment by David-19940718** [comment]
Another question, the teacher network should be pre-trained on a custom dataset first?
